nimbleclient/v1/api/active_directory_memberships.py

```diff
@@ -22,104 +24,131 @@ class ActiveDirectoryMembership(Resource):
-        """
+        '''
         Leaves the Active Directory domain.
 
         Parameters:
         - id       : ID of the active directory.
         - user     : Name of the Activer Directory user with the privilege to leave the domain.
         - password : Password for the Active Directory user.
         - force    : Use this option when there is an error when leaving the domain.
-        """
+        '''
 
         return self.collection.remove(self.id, password, user, force)
 
     def report_status(self):
-        """
+        '''
         Reports the detail status of the Active Directory domain.
 
         Parameters:
         - id : ID of the active directory.
-        """
+        '''
 
         return self.collection.report_status(self.id)
 
     def test_user(self, name):
-        """
+        '''
         Tests whether the user exist in the Active Directory. If the user is present, then the user's group and role information is reported.
 
         Parameters:
         - id   : ID of the Active Directory.
         - name : Name of the Active Directory user.
-        """
+        '''
 
         return self.collection.test_user(self.id, name)
 
     def test_group(self, name):
-        """
+        '''
         Tests whether the user group exist in the Active Directory.
 
         Parameters:
         - id   : ID of the Active Directory.
         - name : Name of the Active Directory group.
-        """
+        '''
 
         return self.collection.test_group(self.id, name)
 
     def delete(self, **kwargs):
         raise NimOSAPIOperationUnsupported("delete operation not supported")
 
+
 class ActiveDirectoryMembershipList(Collection):
     resource = ActiveDirectoryMembership
     resource_type = "active_directory_memberships"
 
-    def remove(self, id, password, user, force=False):
-        """
+    def remove(self,
+               id,
+               password,
+               user,
+               force=False
+               ):
+        '''
         Leaves the Active Directory domain.
 
         Parameters:
         - id       : ID of the active directory.
         - user     : Name of the Activer Directory user with the privilege to leave the domain.
         - password : Password for the Active Directory user.
         - force    : Use this option when there is an error when leaving the domain.
-        """
-
-        return self._client.perform_resource_action(self.resource_type, id, 'remove', id=id, password=password, user=user, force=force)
-
-    def report_status(self, id):
-        """
+        '''
+
+        return self._client.perform_resource_action(self.resource_type, id, 'remove',
+                                                    id=id,
+                                                    password=password,
+                                                    user=user,
+                                                    force=force
+                                                    )
```

Comment:
If you're breaking up the function call like this, there's two choices:

1. Collapse all the parameters to maximize use of the lines and *move the closing parenthesis* to the last line with the parameter:
```python
        return self._client.perform_resource_action(self.resource_type, id, 'remove',
                                                    id=id, password=password, user=user,
                                                    force=force)
```

Or

2. Use a different format that separates each line per parameter:
```python
        return self._client.perform_resource_action(
            self.resource_type, 
            id,
            'remove',
            id=id,
            password=password,
            user=user,
            force=force
        )
```

I like 2 better, but either one is fine. This is the common way of splitting things up.